static/app/components/events/viewHierarchy/index.tsx

```diff
@@ -142,7 +142,7 @@ const Content = styled('div')`
   display: flex;
   flex-direction: row;
   gap: ${space(1)};
-  max-height: 700px;
+  height: 700px;
```

Comment:
Changed from max-height to height because I realized the wireframe would scale a bunch of times as its contents started filling out until we hit max-height.